src/pypromice/postprocess/station_dimensions.csv

```diff
@@ -0,0 +1,35 @@
+stid,station_site,wigos_id,project,station mounting_type,barometer_from_gps,sonic_ranger_from_gps,anometer_from_sonic_ranger,temperature_from_sonic_ranger,height_of_gps_from_station_ground
```

Comment:
I have removed station mounting type from the configuration for now since it is note relevant for BUFR export.
As you also mentioned, it's relevant to address changing station configurations over time.
I will just assume the configurations are static for now.,

Reply:
Sounds good. Can you open an issue so that we can come back to these cases later on?